src/arviz_plots/plots/compareplot.py

```diff
@@ -0,0 +1,190 @@
+    figsize : (float, float), optional
+        If `None`, size is (10, num of models) inches.
+    plot_kwargs : mapping of {str : mapping or False}, optional
+        Valid keys are:
+
+        * point_estimate -> passed to :func:`~.backend.scatter`
+        * error_bar -> passed to :func:`~.backend.line`
+        * ref_line -> passed to :func:`~.backend.line`
+        * shade -> passed to :func:`~.backend.fill_between_y`
+        * labels -> passed to :func:`~.backend.xticks` and :func:`~.backend.yticks`
+        * title -> passed to :func:`~.backend.title`
+        * ticklabels -> passed to :func:`~.backend.yticks`
+
+    pc_kwargs : mapping
+        Passed to :class:`arviz_plots.PlotCollection`
+
+    Returns
+    -------
+    axes :bokeh figure, matplotlib axes or plotly figure
+
+    See Also
+    --------
+    plot_elpd : Plot pointwise elpd differences between two or more models.
+    compare : Compare models based on PSIS-LOO loo or WAIC waic cross-validation.
+    loo : Compute Pareto-smoothed importance sampling leave-one-out cross-validation (PSIS-LOO-CV).
+    waic : Compute the widely applicable information criterion.
+
+    References
+    ----------
+    .. [1] Vehtari et al. (2016). Practical Bayesian model evaluation using leave-one-out
+       cross-validation and WAIC https://arxiv.org/abs/1507.04544
+    """
+    # Check if cmp_df contains the required information criterion
+    information_criterion = ["elpd_loo", "elpd_waic"]
+    column_index = [c.lower() for c in cmp_df.columns]
+    for i_c in information_criterion:
+        if i_c in column_index:
+            break
+    else:
+        raise ValueError(
+            "cmp_df must contain one of the following "
+            f"information criterion: {information_criterion}"
+        )
+
+    # Set default backend
+    if backend is None:
+        backend = rcParams["plot.backend"]
+
+    if plot_kwargs is None:
+        plot_kwargs = {}
+
+    if pc_kwargs is None:
+        pc_kwargs = {}
+
+    # Get plotting backend
+    p_be = import_module(f"arviz_plots.backend.{backend}")
+
+    # Get figure params and create figure and axis
+    pc_kwargs["plot_grid_kws"] = pc_kwargs.get("plot_grid_kws", {}).copy()
+    figsize = pc_kwargs.get("plot_grid_kws", {}).get("figsize", (2000, len(cmp_df) * 200))
+    figsize_units = pc_kwargs.get("plot_grid_kws", {}).get("figsize_units", "dots")
+    chart, target = p_be.create_plotting_grid(1, figsize=figsize, figsize_units=figsize_units)
+
+    # Create plot collection
+    plot_collection = PlotCollection(
+        Dataset({}),
+        viz_dt=DataTree.from_dict(
+            {
+                "/": Dataset(
+                    {"chart": np.array(chart, dtype=object), "plot": np.array(target, dtype=object)}
+                )
+            }
+        ),
+        backend=backend,
+        **pc_kwargs,
+    )
+
+    if isinstance(target, np.ndarray):
+        target = target.tolist()
```

Comment:
It generally is an array of empty lists, so then `.map` subsets the array and target ends up being the list that represents the desired subplot. It is very probably broken for (1,1) "grid" though so we get an empty array instead of an array with a single empty list 🤔